### ISSUE TITLE
Test are not running on CI

### ISSUE THREAD

**Issue by Ashanen** [robolectric/robolectric#5310]
Hi, we are very excited to add Robolectric to our project. I can confirm, that tests are running locally on and off the device but we would like to have something more ( that's why we used Robolectric) and run off device tests on our Team City.

I made a shared tests folder and when invoked manually everything works fine but not on CI. If I invoke testDebugUnitTest task and manually move purposedly my tests to from shared location to Unit Test location they are omitted and not executed neither on CI nor on my local machine. What's the problem? Is there another Gradle task just for Robolectric?

**Comment by Ashanen** [comment]
Any ideas? how do you run Robolectric tests on CI?

**Comment by jgavazzisp** [comment]
Same problem here.

`java.lang.SecurityException: SHA1 digest error for org/bouncycastle/jce/provider/BouncyCastleProvider.class
at sun.security.util.ManifestEntryVerifier.verify(ManifestEntryVerifier.java:223)
at java.util.jar.JarVerifier.processEntry(JarVerifier.java:243)
at java.util.jar.JarVerifier.update(JarVerifier.java:230)
at java.util.jar.JarVerifier$VerifierStream.read(JarVerifier.java:484)
at java.io.FilterInputStream.read(FilterInputStream.java:133)
at java.io.FilterInputStream.read(FilterInputStream.java:107)
at org.robolectric.util.Util.copy(Util.java:46)
at org.robolectric.util.Util.readBytes(Util.java:63)
at org.robolectric.internal.bytecode.SandboxClassLoader.getByteCode(SandboxClassLoader.java:167)
at org.robolectric.internal.bytecode.SandboxClassLoader.maybeInstrumentClass(SandboxClassLoader.java:119)
at org.robolectric.internal.bytecode.SandboxClassLoader.lambda$findClass$0(SandboxClassLoader.java:112)
at org.robolectric.util.PerfStatsCollector.measure(PerfStatsCollector.java:53)
at org.robolectric.internal.bytecode.SandboxClassLoader.findClass(SandboxClassLoader.java:111)
at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
at java.lang.Class.getDeclaredConstructors0(Native Method)
at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
at java.lang.Class.getConstructors(Class.java:1651)
at org.robolectric.util.inject.Injector.findConstructor(Injector.java:249)
at org.robolectric.util.inject.Injector.inject(Injector.java:232)
at org.robolectric.util.inject.Injector.lambda$new$0(Injector.java:115)
at org.robolectric.util.inject.Injector$MemoizingProvider.get(Injector.java:485)
at org.robolectric.util.inject.Injector.getInstanceInternal(Injector.java:213)
at org.robolectric.util.inject.Injector.getInstance(Injector.java:197)
at org.robolectric.util.inject.Injector.getInstance(Injector.java:191)
at org.robolectric.internal.AndroidSandbox.lambda$new$0(AndroidSandbox.java:51)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)`

**Comment by jgavazzisp** [comment]
It happens when gradle plugin is updated to 3.6.0, all tests just fails on CI but works locally.

**Comment by ditn** [comment]
The fix detailed [here](https://github.com/robolectric/robolectric/issues/5299) seems to work.

tldr add `android.jetifier.blacklist=.*bcprov.*` to `gradle.properties`

**Comment by hoisie** [comment]
Closing this issue since it seems to have been fixed in the updated gradle versions. See https://github.com/robolectric/robolectric/issues/5299 for more details.
